policyengine_us/parameters/gov/states/ne/tax/income/credits/eitc/fraction.yaml

```
@@ -1,15 +1,17 @@
-description: NE refundable eitc is this fraction of federal eitc.
+description: Nebraska refundable eitc is this fraction of federal eitc.
```

We don't need to mention refundability since federal EITC is refundable and there aren't different tiers like CDCC
```suggestion
description: Nebraska matches this fraction of the federal earned income tax credit.
```